power/source/main.c

```diff
@@ -0,0 +1,48 @@
+#include <string.h>
+#include <stdio.h>
+#include <time.h>
+
+#include <switch.h>
+
+const char* const chargers[3] = {"None", "Official", "Generic"};
+
+int main(int argc, char **argv)
+{
+    consoleInit(NULL);
+    psmInitialize();
```

Comment:
error checking... (same below)

Reply:
I think this is what you were looking for? It seems a bit overkill, but I suppose it may be possible for any of those calls to individually fail.